test/NuGet.Core.Tests/NuGet.Frameworks.Test/CompatibilityTests.cs

```diff
@@ -904,6 +904,8 @@ public void Compatibility_ProjectCannotInstallGenerationLibraries(string framewo
         [InlineData("xamarinxboxthreesixty")]
         [InlineData("xamarinwatchos")]
         [InlineData("xamarinxboxone")]
+        [InlineData("wasm")]
```

Comment:
Can we have some tests asserting that `wasm` and `monoue` does not support .NET Standard 2.1? Essentially a test verifying the max .NET Standard version is exactly what is expected.

Reply:
We don't do this for any of the other frameworks, and I suspect them to support 2.1 pretty quickly.

I'd love to delegate this information to the framework instead of centralizing it all in one place.